Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM centos:7.2.1511
+
+# gcc for cgo
+RUN yum install -y git g++ \
+		gcc \
+		libc6-dev \
+		make \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV GOLANG_VERSION 1.7rc2
```

Comment:
It seems that the latest Go version is **1.7.1** (https://golang.org/dl/), but **1.7** is great too. We can change it later.